cpp/data/sql_updates/vufind.2

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE IF NOT EXISTS vufind.tuefind_rss_feeds (
+    id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+    feed_name VARCHAR(200) NOT NULL,
+    subsystem_types SET('krimdok', 'ixtheo', 'relbib') NOT NULL,
+    feed_url VARCHAR(1000) NOT NULL,
+    website_url VARCHAR(1000) NOT NULL,
+    title_suppression_regex VARCHAR(200) DEFAULT NULL,
+    strptime_format VARCHAR(50) DEFAULT NULL,
+    downloader_time_limit INT NOT NULL DEFAULT 30,
+    CONSTRAINT id_constraint UNIQUE (id),
+    CONSTRAINT feed_name_constraint UNIQUE (feed_name),
+    CONSTRAINT feed_url_constraint UNIQUE (feed_url(768)),
+    CONSTRAINT website_url_constraint UNIQUE (website_url(768))
+) DEFAULT CHARSET=utf8mb4 COLLATE utf8mb4_bin;
+
+
+CREATE TABLE IF NOT EXISTS vufind.tuefind_rss_items (
+    rss_feeds_id INT UNSIGNED NOT NULL,
```

Comment:
Please add missing id columns to tuefind_rss_items and tuefind_rss_subscriptions, see ubtue/tuefind#1403

Reply:
vufind.tuefind_rss_feeds has an id column already.  I have added an id column to vufind.tuefind_rss_items.